rmake.py

```diff
@@ -190,22 +213,24 @@ def config_cmd():
     if args.static_lib:
         cmake_options.append( f"-DBUILD_SHARED_LIBS=OFF" )
 
-    if args.skip_ld_conf_entry:
+    if args.relocatable:
+        rocm_rpath = os.getenv( 'ROCM_RPATH', "/opt/rocm/lib:/opt/rocm/lib64")
+        cmake_options.append( f'-DCMAKE_SHARED_LINKER_FLAGS=" -Wl,--enable-new-dtags -Wl,--rpath,{rocm_rpath}"' )
+
+    if args.skip_ld_conf_entry or args.relocatable:
         cmake_options.append( f"-DROCM_DISABLE_LDCONFIG=ON" )
```

Comment:
This may need to change once the PR goes in changing install.sh but matching fine for now